SamplesDashboard/SamplesDashboard/ClientApp/src/types/samples.ts

```diff
@@ -10,9 +10,12 @@ export interface IRepositoryItem {
     issues: any;
     stargazers: any;
     forks: any;
-    url: string;
+    url: any;
```

Comment:
>any [](start = 9, length = 3)

do we really need to use any here.  Every time i see these it's like losing the value of using typescript.

Reply:
i meant the code for the owner portion.  I'm not sure why the change has been made here for url

---
In reply to: [404311080](https://github.com/microsoftgraph/msgraph-samples-dashboard/pull/105#discussion_r404311080) [](ancestors = 404311080)

Reply:
I'll change it to be of type string